lib/js/tests/Webapi/Webapi__Dom/Webapi__Dom__Window__test.js

```diff
@@ -16,7 +17,7 @@ window.crypto;
 
 window.document;
 
-window.frameElement;
+Caml_option.nullable_to_opt(window.frameElement);
```

Comment:
Hmm, this looks fishy. Are you using a really old version of BuckleScript here?

Reply:
The newest, as far as I know–6.0.0-dev.1. The handling of `[@bs.return nullable]` might have changed slightly.

Reply:
Might be better to use 5.x.x for this since the 6-series is pre-release, just in case there's any differences. But the generation does look right. `Caml_option` didn't exist until very recently either it seems.

Reply:
OK, I've locally used v5.0.2 (the latest released version) and the `Caml_option` stuff is gone now.

Reply:
Oh, interesting, I didn't realize it was that new. Might even be a regression then.